content/blog/2018-05-09-onboarding-social-weather.md

```diff
@@ -0,0 +1,672 @@
+  head() %>%
+  knitr::kable()
+```
+
+| line\_id                                                                                               | word     |
+|:-------------------------------------------------------------------------------------------------------|:---------|
+| rrlite karthik 2015-04-12 20:56:04 - \] add a ropensci footer.                                         | add      |
+| rrlite karthik 2015-04-12 20:56:04 - \] add a ropensci footer.                                         | a        |
+| rrlite karthik 2015-04-12 20:56:04 - \] add a ropensci footer.                                         | ropensci |
+| rrlite karthik 2015-04-12 20:56:04 - \] add a ropensci footer.                                         | footer   |
+| rrlite karthik 2015-04-12 20:56:04 - \] add an appropriate entry into ropensci.org/packages/index.html | add      |
+| rrlite karthik 2015-04-12 20:56:04 - \] add an appropriate entry into ropensci.org/packages/index.html | an       |
+
+Then, we can compute the correlation.
+
+``` r
+word_cors <- onboarding_line_words %>%
+  dplyr::group_by(word) %>%
+  dplyr::filter(!word %in% stopwords) %>%
+  dplyr::filter(n() >= 20) %>%
+  widyr::pairwise_cor(word, line_id, sort = TRUE)
+```
+
+For instance, what often goes in the same line as vignette?
+
+``` r
+dplyr::filter(word_cors, item1 == "vignette")
+```
+
+    ## # A tibble: 853 x 3
+    ##    item1    item2     correlation
+    ##    <chr>    <chr>           <dbl>
+    ##  1 vignette readme         0.176 
+    ##  2 vignette vignettes      0.174 
+    ##  3 vignette chunk          0.145 
+    ##  4 vignette eval           0.120 
+    ##  5 vignette examples       0.108 
+    ##  6 vignette overview       0.0933
+    ##  7 vignette building       0.0914
+    ##  8 vignette link           0.0863
+    ##  9 vignette maps           0.0840
+    ## 10 vignette package        0.0831
+    ## # ... with 843 more rows
+
+Now let’s plot the network of these relationships between words, using
+the [`igraph`](http://igraph.org/r/) and
+[`ggraph`](https://github.com/thomasp85/ggraph) packages, by
+respectively Gábor Csárdi and Támas Nepusz, and Thomas Lin Pedersen.
+
+``` r
+library("igraph")
+library("ggraph")
+
+set.seed(2016)
+
+word_cors %>%
+  dplyr::filter(correlation > .35) %>%
+  graph_from_data_frame() %>%
+  ggraph(layout = "fr") +
+  geom_edge_link(aes(edge_alpha = correlation), show.legend = FALSE) +
+  geom_node_point(color = "lightblue", size = 5) +
+  geom_node_text(aes(label = name), repel = TRUE) +
+  theme_void()
+```
+
+![](/img/blog-images/2018-05-09-onboarding-social-weather/unnamed-chunk-14-1.png)
+
+This figure gives a good sample of things discussed in reviews. Despite
+our efforts filtering words specific to issues, some of them remain very
+specific, such as country/city/location that are very frequent in
+`ropenaq` review.
+
+#### How positive is onboarding?
+
+Using sentiment analysis, we can look how positive lines are.
```

Comment:
... look at how positive lines are.

Do you mean review comments (in lines?). To keep the story crystal clear, need to keep the language clearly referring to onboarding review components rather than the elements they've been extracted into for analysis.